run_bot_simple.py

```
@@ -0,0 +1,85 @@
+
+"""
+Standalone script to run the Native AI Telegram bot
+This script uses a non-asyncio approach to avoid event loop conflicts
+"""
+
+import os
+import sys
+import logging
+from telegram.ext import ApplicationBuilder
+from langchain_openai import ChatOpenAI
```

Ruff (0.12.2)

`langchain_openai.ChatOpenAI` imported but unused

Remove unused import: `langchain_openai.ChatOpenAI`

(F401)